message.go

```diff
@@ -315,6 +315,11 @@ func (cli *Client) handleHistorySyncNotification(notif *waProto.HistorySyncNotif
 			go cli.handleHistoricalPushNames(historySync.GetPushnames())
 		} else if len(historySync.GetConversations()) > 0 {
 			go cli.storeHistoricalMessageSecrets(historySync.GetConversations())
+
+			if historySync.GetSyncType() == waProto.HistorySync_INITIAL_BOOTSTRAP ||
```

Comment:
Here I was not sure, HistorySync_INITIAL_BOOTSTRAP & waProto.HistorySync_RECENT(chunk-order-1) is the only cases we should store the last message

Reply:
I removed the message storage entirely since it's not necessary